### ISSUE TITLE
Some possibly incorrect timex extractions

### ISSUE THREAD

**Issue by bgyori** [clulab/eidos#720]
In c51cf0a4cb520574d1e6f464224a175b.jsonld of 20191011-DART.zip there are some examples of time expressions where the text looks normal like "2011" but the start and end dates are not as expected. Below are two examples with line numbers corresponding to the original file. Any thoughts?

``` 232379       "timexes" : [ {                                                           
 232380         "@type" : "TimeExpression",                                             
 232381         "@id" : "_:TimeExpression_25",                                          
 232382         "startOffset" : 64111,                                                  
 232383         "endOffset" : 64115,                                                    
 232384         "text" : "2011",                                                        
 232385         "intervals" : [ {                                                       
 232386           "@type" : "TimeInterval",                                             
 232387           "@id" : "_:TimeInterval_14",                                          
 232388           "start" : "0011-01-01T00:00",                                         
 232389           "end" : "0012-01-01T00:00"                                            
 232390         } ]                                                                     
```

```
 414333       "timexes" : [ {                                                           
 414334         "@type" : "TimeExpression",                                             
 414335         "@id" : "_:TimeExpression_35",                                          
 414336         "startOffset" : 113915,                                                 
 414337         "endOffset" : 113919,                                                   
 414338         "text" : "2011",                                                        
 414339         "intervals" : [ {                                                       
 414340           "@type" : "TimeInterval",                                             
 414341           "@id" : "_:TimeInterval_16",                                          
 414342           "start" : "0001-01-01T00:00",                                         
 414343           "end" : "0002-01-01T00:00"                                            
 414344         } ]                                                                     
 414345       } ]                                                                       
```

**Comment by kwalcock** [comment]
I've been looking into it...

**Comment by EgoLaparra** [comment]
I'm also trying to figure out what is going on with these examples.

**Comment by kwalcock** [comment]
Texts:
```
At a critical moment in 2011 the factory producing RUTF in Ethiopia had an issue with hygiene and had to be closed for a week , affecting supply critically .
```
and
```
Whilst the 2011 operation saved lives , it did not save livelihoods .
```
although remove a few spaces because they have been tokenized.

**Comment by kwalcock** [comment]
So
```
At a critical moment in 2011 the factory producing RUTF in Ethiopia had an issue with hygiene and had to be closed for a week, affecting supply critically.
```
and
```
Whilst the 2011 operation saved lives it did not save livelihoods.
```
It seems like they work OK in isolation.

**Comment by kwalcock** [comment]
I have been able to reproduce by processing the entire document.  Details shortly...

**Comment by kwalcock** [comment]
It seems to be reproducible on the master branch and when processing the entire file.  I'm using ExtractFromDirectory and just processing this:

[c51cf0a4cb520574d1e6f464224a175b.txt](https://github.com/clulab/eidos/files/3941251/c51cf0a4cb520574d1e6f464224a175b.txt)

Is that enough of a start, @EgoLaparra?

**Comment by EgoLaparra** [comment]
The error is produced if there are consecutive control characters, like `^C` or `^L` anywhere in the text before the time expression. The timeparser needs to replace them internally, but it does it with a single character resulting in wrong character offsets. I will fix this but, meanwhile, the c51cf0a4cb520574d1e6f464224a175b.txt can be parsed removing the consecutive   `^C` and `^L` in lines 779, 963 and 4333.

**Comment by kwalcock** [comment]
This has been addressed with https://github.com/clulab/timenorm/pull/52 that is published in timenorm 1.0.3.
